README.md

```diff
@@ -214,6 +214,73 @@ Note: When using a TPM device through the Linux kernel driver make sure sufficie
 sudo adduser yourusername tss
 ```
 
+#### With QEMU and swtpm
+
+This demonstrates using wolfTPM in QEMU to communicate using the linux
+kernel device "/dev/tpmX". You will need to install or build
+[swtpm](https://github.com/stefanberger/swtpm). Below are a short
+method to build. You may need to consult the instructions for
+[libtpms](https://github.com/stefanberger/libtpms/wiki#compile-and-install-on-linux)
+and
+[swtpm](https://github.com/stefanberger/swtpm/wiki#compile-and-install-on-linux)
+
+```
+PREFIX=$PWD/inst
+git clone git@github.com:stefanberger/libtpms.git
+cd libtpms/
+./autogen.sh --with-openssl --with-tpm2 --prefix=$PREFIX && make install
+cd ..
+git clone git@github.com:stefanberger/swtpm.git
+cd swtpm
+PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig/ ./autogen.sh --with-openssl --with-tpm2 --prefix=$PREFIX && make install
+cd ..
+```
+
+You can setup a basic linux installation. Other installation bases can
+be used. This step will take some time to install the base linux
+system.
+
+```
+# download mini install image
+curl -O http://archive.ubuntu.com/ubuntu/dists/bionic-updates/main/installer-amd64/current/images/netboot/mini.iso
+# create qemu image file
+qemu-img create -f qcow2 lubuntu.qcow2 5G
+# start swtpm and start qemu with install media
+swtpm socket --tpm2 --tpmstate dir=/tmp/mytpm1 --ctrl type=unixio,path=/tmp/mytpm1/swtpm-sock --log level=20 &
```

Comment:
consider adding relative path inst/bin/swtpm OR export PATH=$PATH:$PWD/inst/bin

Reply:
Good idea. I reworked to use the `PREFIX` environment variable set above.